### PR TITLE
fix documentation error: volume-opt in service create

### PR DESCRIPTION
**Pull request by pacoxu**
Signed-off-by: pacoxu<paco.xu@daocloud.io>

> docker service create --replicas 1 --mount type=volume,source=test2,target=/tmp,volume-driver=rexray,volume-nocopy=true,volume-opt=size=8,volume-opt=volume-type=defaultSP,volume-opt=size=24 --mount  type=volume,source=test4,target=/tmp1,volume-driver=rexray  --env MYVAR=foo daocloud.io/daocloud/dao-2048:latest

As the example above shows, correct opts for volume are like 'volume-opt=volume-type=defaultSP,volume-opt=size=24'.

### CODE REVIEW

File: docs/reference/commandline/service_create.md [hunk 1]
```diff
@@ -424,7 +424,7 @@ The following options can only be used for named volumes (`type=volume`);
       Options specific to a given volume driver, which will be passed to the
       driver when creating the volume. Options are provided as a comma-separated
       list of key/value pairs, for example,
-      <tt>volume-opt=some-option=some-value,some-other-option=some-other-value</tt>.
+      <tt>volume-opt=some-option=some-value,volume-opt=some-other-option=some-other-value</tt>.
```
Comment (mstanleyjones) [inline docs/reference/commandline/service_create.md]: Is this still not quite correct? I think the other options like this, it is like `--volume-opt key=value --volume-opt otherkey=othervalue`

Comment (pacoxu) [inline docs/reference/commandline/service_create.md]: @mstanleyjones 

> docker service create \
--replicas 1 \
--mount type=volume,source=test2,target=/tmp,volume-driver=rexray,volume-nocopy=true,volume-opt=size=8,volume-opt=volume-type=defaultSP,volume-opt=size=24 \
--mount  type=volume,source=test4,target=/tmp1,volume-driver=rexray  \
--env MYVAR=foo daocloud.io/daocloud/dao-2048:latest 

For service creation, --volume-opt  is not an option. It is one option of "--mount".
You can see the example above.

**Comment by pacoxu** [comment]
@mstanleyjones you can look into the mount options @ https://github.com/docker/docker/blob/master/opts/mount.go#L111

as shown below, valid options are [ type, source, target, readonly, consistency, bind-propagation, 
 volume-nocopy, volume-driver, volume-label, volume-opt, tmpfs-size, tmpfs-mode]. 
image

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**